TestTshirtsSize.cpp

```diff
@@ -0,0 +1,28 @@
+// TestTshirtsSize.cpp
+#include <gtest/gtest.h>
+#include "TshirtsSize.h"
+
+TEST(SizeTest, HandlesSmallSize) {
+    EXPECT_EQ(tshirtsSize(37), 'S');
+}
+
+TEST(SizeTest, HandlesMediumSizeLowerBoundary) {
+    EXPECT_EQ(tshirtsSize(38), 'M');

```

Comment:
Size 38 is made `M` over here. What is the reason you did not expect it to be `S`?

Reply:
The Reason I considered 38 to be medium. As looking at the logic intention, looks like between 38 to 42 size is medium. As the code is buggy it is not considering equal to condition.

Reply:
Maybe `M` is correct, but I am more interested in the reasoning.
The above reasoning is done by looking at the code. Can you think of the customer experience instead?